### PR TITLE
ableset 2.7.1 (new cask)

### PR DESCRIPTION
**Pull request by JudahZF**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/a/ableset.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+cask "ableset" do
+  arch arm: "aarch64", intel: "x86_64"
+  folder = on_arch_conditional arm: "-arm64", intel: ""
```
Comment (khipp) [inline Casks/a/ableset.rb:3]:
```suggestion
  arch arm: "-arm64"
```

We don’t need to specify the `intel:` parameter if we are setting it to an empty string.

File: Casks/a/ableset.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+  version "2.7.1"
+  sha256 arm:   "27e7bd8d55b474c92589b79c6272269151a9941de82969698338d9c2d332a3a7",
+         intel: "6c312f3b5cf08943fd837eb8a5f81d8fc0e8ede592bb99c18b423f7a400ce084"
+
+  url "https://ableset.app/download/mac#{folder}/#{version}"
```
Comment (khipp) [inline Casks/a/ableset.rb:9]:
```suggestion
  url "https://s3.amazonaws.com/download.ableset.app/mac#{arch}/AbleSet-#{version}#{arch}.dmg",
      verified: "s3.amazonaws.com/download.ableset.app/"
```

File: Casks/a/ableset.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  name "AbleSet"
+  desc "Ableton setlist manager"
+  homepage "https://ableset.app/"
+
+  livecheck do
+    url "https://ableset.app/download/mac/"
+    strategy :header_match
```
Comment (khipp) [inline Casks/a/ableset.rb:16]:
```suggestion
    url "https://s3.amazonaws.com/download.ableset.app/mac#{arch}/latest-mac.yml"
    strategy :electron_builder
```

We can use the feed of the in-app updater with the `ElectronBuilder` strategy.

File: Casks/a/ableset.rb [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  end
+
+  depends_on macos: ">= :high_sierra"
```
Comment (khipp) [inline Casks/a/ableset.rb:19]:
```suggestion
  auto_updates true
  depends_on macos: ">= :high_sierra"
```

**Review by khipp — COMMENTED** [review]
Thank you for your first contribution to `homebrew-cask`, @JudahZF! 🎉

I left a few suggestions below. Please also try to squash your commits.

File: Casks/a/ableset.rb (hunk 1 above)
Comment (JudahZF) [reply Casks/a/ableset.rb]: Changed in latest push

File: Casks/a/ableset.rb (hunk 2 above)
Comment (JudahZF) [reply Casks/a/ableset.rb]: Fixed in latest commit

File: Casks/a/ableset.rb (hunk 3 above)
Comment (JudahZF) [reply Casks/a/ableset.rb]: Changed in latest commit

**Review by khipp — APPROVED** [review]
Thank you!

**Pull request merged**